File: FitnessCentar/FitnessCenatr.Model/Komentari.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitnessCentar.Model
{
    public class Komentari
    {
        public int id { get; set; }

        public int? NovostId { get; set; }

        public int? KorisnikId { get; set; }

        public string? Tekst { get; set; }

        public DateTime? DatumKomentara { get; set; }

       
    }
}

File: FitnessCentar/UI/fitness_mobile/lib/models/komentari.dart


import 'package:json_annotation/json_annotation.dart';

part 'komentari.g.dart';

@JsonSerializable()
class Komentari{
int? id;
int? novostId;
int? korisnikId;
String? tekst;
DateTime? datumKomentara;


Komentari(this.id,this.korisnikId,this.novostId,this.tekst,this.datumKomentara);

  factory Komentari.fromJson(Map<String, dynamic> json) =>
      _$KomentariFromJson(json);

  Map<String, dynamic> toJson() => _$KomentariToJson(this);


}

